build-tools/create-vsix/source.extension.vsixmanifest

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+	<Metadata>
+		<Identity Id="4489ebd9-10e4-42d0-bc3c-0639c608aaea" Version="1.0" Language="en-US" Publisher="Company" />

```

Comment:
Let's make these match the values in https://github.com/xamarin/XamarinVS/blob/master/src/Installer/Android.Sdk/source.extension.vsixmanifest#L4